scan/lib/scan/runner.rb

```diff
@@ -101,6 +93,26 @@ def handle_results(tests_exit_status)
       end
     end
 
+    def output_simulator_logs
```

Comment:
maybe name it `include_simulator_logs` instead.

Output makes me think they will be pasted to the output.